packages/agoric-cli/src/commands/oracle.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable @jessie.js/no-nested-await */
+// @ts-check
+/* eslint-disable func-names */
+/* global fetch, process */
+import { Command } from 'commander';
+import { inspect } from 'util';
+import { makeAmountFormatter } from '../lib/format.js';
+import {
+  boardSlottingMarshaller,
+  makeRpcUtils,
+  storageHelper,
+} from '../lib/rpc.js';
+
+const { agoricNames, fromBoard, vstorage } = await makeRpcUtils({ fetch });
+
+/**
+ *
+ * @param {import('anylogger').Logger} logger
+ */
+export const makeOracleCommand = async logger => {
```

Comment:
I'm still hesitant to approve this exception to our style.

`makeOracleCommand` closes over (the output of) `fetch` (by way of `agoricNames` etc.).

> No modules should export powerful objects (for example, objects that close over `fs`). -- https://github.com/Agoric/agoric-sdk/wiki/OCap-Discipline